project/load.sbt

```diff
@@ -0,0 +1,13 @@
+libraryDependencies ++= {
+  val profile = System.getProperty("sbt.profile")
+  val dir = new File(System.getProperty("user.dir"))
+  val buildFile = dir / s"build.sbt.$profile"
+
+  if (buildFile.exists) {
+    IO.copyFile(buildFile, dir / "build.dynamic.sbt")
+  } else {
+    sys.error(s"""Unknown profile "$profile"; specify a supported one, e.g. sbt -Dsbt.profile=cross-publish""")
+  }
```

Comment:
I think we should see if there's any feedback from the tooling team, otherwise we can always merge this PR and modify as per their suggestion with future PRs.

Reply:
Got their help - we're crossbuilt for 2.11 and 2.12 the proper way now.